NOTICE.txt

```diff
@@ -2262,7 +2262,7 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 ---------------------------------------------------------
 
-jmespath 0.10.0 - MIT
+jmespath-community 1.1.0 - MIT
```

Comment:
Does the [copyright of `jmespath-community`](https://github.com/jmespath-community/python-jmespath/blob/main/LICENSE) still belong to Amazon?

Reply:
We have forked the repository, so I’m not sure what is the best practice.
Also, I’m not familiar at all with respect to licenses and copyright 😱.

What do you think in your experience ?